downstream/modules/platform/proc-new-aap-instance-upgrade.adoc

```diff
@@ -0,0 +1,31 @@
+[id="proc-new-aap-instance-upgrade_{context}"]
+
+= Deploy a new instance for a side-by-side upgrade
+
+To proceed with the side-by-side upgrade process, deploy a second instance of Ansible Tower 3.8.x with the same instance groups configurations. This new instance will receive the content and configuration from your original instance, and will later be upgraded to {PlatformName} {PlatformVers}.
+
+== Deploy a new instance of Ansible Tower
+
+Start by downloading the Tower installer version that matches your original Tower instance version, then running the installer:
+
+. Download the Tower installer version that matches your original Tower instance by navigating to this link: https://releases.ansible.com/ansible-tower/setup/
+. Navigate to the installer, then open the `inventory` file using a text editor to configure the `inventory` file for a Tower installation:
+.. In addition to any Tower configurations, remove any fields containing `isolated_group` or `instance_group`.
++
+NOTE: For more information about installing Tower using the {PlatformNameShort} installer, see the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_installation_guide/index[{PlatformNameShort} Installation Guide] for your specific installation scenario.
+. Run the `setup.sh` script to begin the installation.
+
+With the new instance now installed, you will need to configure the Tower settings to match the instance groups from your original Tower instance.
+
+== Recreate instance groups in the new instance
+
+Log in to your new instance of Ansible Tower and configure the instance groups to match the same instance group configuration as your old instance.
+
+NOTE: Make note of all instance groups from your original Tower instance. You will need to recreate these groups in your new instance.
+
+. Log in to your new instance of Tower.
+. On the side panel, navigate to *Administration* > *Instance groups*.
+. Recreate instance groups from your original instance:
```

Comment:
I don't think you need this statement since this is the procedure for recreating the instances. I would delete this sentence and continue with original steps with Click *+* > *Create instance group* and this would be step 3. No sub-steps needed.